datadog-monitors-monitor-handler/docs/monitorstate-groups.md

```diff
@@ -1,4 +1,4 @@
-# Datadog::Monitors::Monitor State Groups
+# Datadog::Monitors::Monitor MonitorState Groups
```

Comment:
Are these changed as part of the resource migration, or just the doc generation wasn't up to date?

Reply:
I think doc wasn't up to date with model